tools/schematics/dt-component/files/libs/barista-components/__name@dasherize__/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("//tools/bazel_rules:index.bzl", "stylelint")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "<%= dasherize(name) %>",
+    srcs = glob(
+        include = ["**/*.ts"],
+        exclude = [
+            "**/*.spec.ts",
+            "src/test-setup.ts",
+        ],
+    ) + glob([
+        "**/*.html",
+        "**/*.scss",
+    ]),
+)
```

Comment:
I would add a small setup for typescript like the button.

Reply:
Added it, thanks.